bin/packages/build.js

```diff
@@ -193,7 +193,14 @@ if ( files.length ) {
 	bar.tick( 0 );
 
 	stream = glob.stream(
-		[ `${ PACKAGES_DIR }/*/src/**/*.js`, `${ PACKAGES_DIR }/*/src/*.scss` ],
+		[
+			`${ PACKAGES_DIR }/*/src/**/*.js`,
+			`${ PACKAGES_DIR }/*/src/*.scss`,
+			`${ PACKAGES_DIR }/block-library/src/**/*.js`,
+			`${ PACKAGES_DIR }/block-library/src/*/style.scss`,
+			`${ PACKAGES_DIR }/block-library/src/*/editor.scss`,
+			`${ PACKAGES_DIR }/block-library/src/*.scss`,
```

Comment:
You'd think so... But for some reason it doesn't work without those 2 lines. In fact that thing set me back days, I couldn't get the compiling to work until I added those seemingly unnecessary lines :shrug:

Reply:
Maybe the order of these glob patterns matters? 🤷

Reply:
I just tried all combos I could think of now and the current implementation - however weird it may look - seems to be the only thing that properly compiles. 
Huge props to @herregroen, he was the one that helped me figure this out and I was able to continue work on the implementation :heart: